### PR TITLE
Cloudflare Stream customer-specific subdomains

### PR DESCRIPTION
**Pull request by irvinebroque**
refs https://jira.cfops.it/browse/PCX-4292

- Replaces all instances of `videodelivery.net` and `cloudflarestream.com` with `customer-<CODE>.cloudflarestream.com` (done carefully instead of just a find-and-replace, since `watch.videodelivery.net` uses `customer-<CODE>.cloudflarestream.com/<VIDEO_UID>/watch` when used with the customer-specific subdomain)
- Does not change URLs in examples for direct creator uploads, because these URLs remain unchanged in our API (see https://jira.cfops.it/browse/STREAM-4364)
- Adds additional language to clarify how to allow _only_ videos from your own Cloudflare Account to be played on your website, using Content Security Policy. (refs https://jira.cfops.it/browse/PCX-4414)
- Adds changelog entry for customer-specific subdomains feature (refs https://jira.cfops.it/browse/PCX-4413)

In addition, this PR also makes the following changes across all Stream documentation:

- Ensures that we always refer to video UIDs and live input UIDs, rather than a combination of ID and UID, for clarity and consistency with what fields in our API reference. (ex: `VIDEO_UID` instead of a mix of `VIDEOID`, `VIDEO_ID`, and `VIDEO_UID`)
- Ensures that we always use `<>` to denote values that the reader needs to replace. Previously, we used a mix of `{}`, `$` and `<>` for this. (ex: always `ACCOUNT_ID` instead of a mix of `{account_id}`, `<ACCOUNT_ID>` and `$ACCOUNT`
- Replaces a few lingering instances of API Keys with API token examples.

### CODE REVIEW

**Review by dcpena — APPROVED** [review]

File: content/stream/viewing-videos/securing-your-stream.md [hunk 1]
```diff
@@ -83,13 +83,13 @@ To render the video, insert the `token` value in place of the `video id`:
...
+<iframe src="https://customer-<CODE>.iframe.cloudflarestream.com/eyJhbGciOiJSUzI1NiIsImtpZCI6ImNkYzkzNTk4MmY4MDc1ZjJlZjk2MTA2ZDg1ZmNkODM4In0.eyJraWQiOiJjZGM5MzU5ODJmODA3NWYyZWY5NjEwNmQ4NWZjZDgzOCIsImV4cCI6IjE2MjE4ODk2NTciLCJuYmYiOiIxNjIxODgyNDU3In0.iHGMvwOh2-SuqUG7kp2GeLXyKvMavP-I2rYCni9odNwms7imW429bM2tKs3G9INms8gSc7fzm8hNEYWOhGHWRBaaCs3U9H4DRWaFOvn0sJWLBitGuF_YaZM5O6fqJPTAwhgFKdikyk9zVzHrIJ0PfBL0NsTgwDxLkJjEAEULQJpiQU1DNm0w5ctasdbw77YtDwdZ01g924Dm6jIsWolW0Ic0AevCLyVdg501Ki9hSF7kYST0egcll47jmoMMni7ujQCJI1XEAOas32DdjnMvU8vXrYbaHk1m1oXlm319rDYghOHed9kr293KM7ivtZNlhYceSzOpyAmqNFS7mearyQ" style="border: none;" height="720" width="1280" allow="accelerometer; gyroscope; autoplay; encrypted-media; picture-in-picture;" allowfullscreen="true"></iframe>
```
Comment (bkrebsbach) [inline content/stream/viewing-videos/securing-your-stream.md]: This (and anywhere else iframe is used) should be `customer-<CODE>.cloudflarestream.com/<VIDEO_ID>/iframe`

File: content/stream/viewing-videos/securing-your-stream.md [hunk 2]
```diff
@@ -272,21 +270,21 @@ If you are using the Stream Player, insert the token returned by the Worker in S
...
```
Comment (bkrebsbach) [inline content/stream/viewing-videos/securing-your-stream.md]: As above `customer-<CODE>.cloudflarestream.com/<VIDEO_ID>/iframe`

File: content/stream/viewing-videos/securing-your-stream.md [hunk 3]
```diff
@@ -272,21 +270,21 @@ If you are using the Stream Player, insert the token returned by the Worker in S
...
 
-`https://videodelivery.net/eyJhbGciOiJSUzI1NiIsImtpZCI6ImNkYzkzNTk4MmY4MDc1ZjJlZjk2MTA2ZDg1ZmNkODM4In0.eyJraWQiOiJjZGM5MzU5ODJmODA3NWYyZWY5NjEwNmQ4NWZjZDgzOCIsImV4cCI6IjE2MjE4ODk2NTciLCJuYmYiOiIxNjIxODgyNDU3In0.iHGMvwOh2-SuqUG7kp2GeLXyKvMavP-I2rYCni9odNwms7imW429bM2tKs3G9INms8gSc7fzm8hNEYWOhGHWRBaaCs3U9H4DRWaFOvn0sJWLBitGuF_YaZM5O6fqJPTAwhgFKdikyk9zVzHrIJ0PfBL0NsTgwDxLkJjEAEULQJpiQU1DNm0w5ctasdbw77YtDwdZ01g924Dm6jIsWolW0Ic0AevCLyVdg501Ki9hSF7kYST0egcll47jmoMMni7ujQCJI1XEAOas32DdjnMvU8vXrYbaHk1m1oXlm319rDYghOHed9kr293KM7ivtZNlhYceSzOpyAmqNFS7mearyQ/manifest/video.m3u8`
+`https://customer-<CODE>.iframe.cloudflarestream.com/eyJhbGciOiJSUzI1NiIsImtpZCI6ImNkYzkzNTk4MmY4MDc1ZjJlZjk2MTA2ZDg1ZmNkODM4In0.eyJraWQiOiJjZGM5MzU5ODJmODA3NWYyZWY5NjEwNmQ4NWZjZDgzOCIsImV4cCI6IjE2MjE4ODk2NTciLCJuYmYiOiIxNjIxODgyNDU3In0.iHGMvwOh2-SuqUG7kp2GeLXyKvMavP-I2rYCni9odNwms7imW429bM2tKs3G9INms8gSc7fzm8hNEYWOhGHWRBaaCs3U9H4DRWaFOvn0sJWLBitGuF_YaZM5O6fqJPTAwhgFKdikyk9zVzHrIJ0PfBL0NsTgwDxLkJjEAEULQJpiQU1DNm0w5ctasdbw77YtDwdZ01g924Dm6jIsWolW0Ic0AevCLyVdg501Ki9hSF7kYST0egcll47jmoMMni7ujQCJI1XEAOas32DdjnMvU8vXrYbaHk1m1oXlm319rDYghOHed9kr293KM7ivtZNlhYceSzOpyAmqNFS7mearyQ/manifest/video.m3u8`
```
Comment (bkrebsbach) [inline content/stream/viewing-videos/securing-your-stream.md]: Same, `customer-<CODE>.cloudflarestream.com/<VIDEO_ID>/iframe`

File: content/stream/viewing-videos/using-the-player-api.md [hunk 4]
```diff
@@ -13,7 +13,7 @@ The SDK provides an API for programmatically controlling the player and listenin
 ```html
 <!-- You can use styles and CSS on this iframe element where the video player will appear -->
 <iframe
-  src="https://iframe.videodelivery.net/$VIDEOID"
+  src="https://customer-<CODE>.iframe.cloudflarestream.com/<VIDEO_UID>"
```
Comment (bkrebsbach) [inline content/stream/viewing-videos/using-the-player-api.md]: `customer-<CODE>.cloudflarestream.com/<VIDEO_ID>/iframe`

File: content/stream/viewing-videos/using-the-stream-player.md [hunk 5]
```diff
@@ -6,11 +6,14 @@ weight: 1
...
-The Stream player can be placed on a web page in an iframe element with the video UID (or [signed token](/stream/viewing-videos/securing-your-stream/)) replacing `$VIDEOID` in the example below.
+To add the Stream Player to a web page, you can either:
+
+1. Generate an embed code in the [Stream Dashboard](https://dash.cloudflare.com/?to=/:account/stream) for a specific video or live input.
+2. Use the code example below, replacing `<VIDEO_UID>` with the video UID (or [signed token](/stream/viewing-videos/securing-your-stream/)) and `<CODE>` with the your unique customer code, which can be found in the [Stream Dashboard](https://dash.cloudflare.com/?to=/:account/stream).
 
 ```html
 <iframe
-  src="https://iframe.videodelivery.net/$VIDEOID"
+  src="https://customer-<CODE>.iframe.cloudflarestream.com/<VIDEO_UID>"
```
Comment (bkrebsbach) [inline content/stream/viewing-videos/using-the-stream-player.md]: customer-<CODE>.cloudflarestream.com/<VIDEO_ID>/iframe

File: content/stream/viewing-videos/using-the-stream-player.md [hunk 6]
```diff
@@ -29,7 +32,7 @@ Changing the `height` and `width` attributes on the `iframe` will change the pix
...
```
Comment (bkrebsbach) [inline content/stream/viewing-videos/using-the-stream-player.md]: customer-<CODE>.cloudflarestream.com/<VIDEO_ID>/iframe

File: content/stream/viewing-videos/using-the-stream-player.md [hunk 7]
```diff
@@ -46,7 +49,7 @@ To make an iframe responsive, it needs styles to enforce an aspect ratio by sett
 <!-- padding-top calculation is height / width (assuming 16:9 aspect ratio) -->
 <div style="position: relative; padding-top: 56.25%">
   <iframe
-    src="https://iframe.videodelivery.net/$VIDEOID"
+    src="https://customer-<CODE>.iframe.cloudflarestream.com/<VIDEO_UID>"
```
Comment (bkrebsbach) [inline content/stream/viewing-videos/using-the-stream-player.md]: customer-<CODE>.cloudflarestream.com/<VIDEO_ID>/iframe

File: content/stream/viewing-videos/using-the-stream-player.md [hunk 8]
```diff
@@ -58,7 +61,7 @@ To make an iframe responsive, it needs styles to enforce an aspect ratio by sett
 
 Player options are configured with querystring parameters in the iframe's `src` attribute. For example:
 
-`https://iframe.videodelivery.net/$VIDEOID?autoplay=true&muted=true`
+`https://customer-<CODE>.iframe.cloudflarestream.com/<VIDEO_UID>?autoplay=true&muted=true`
```
Comment (bkrebsbach) [inline content/stream/viewing-videos/using-the-stream-player.md]: customer-<CODE>.cloudflarestream.com/<VIDEO_ID>/iframe

File: content/stream/viewing-videos/using-the-stream-player.md [hunk 9]
```diff
@@ -118,7 +121,9 @@ The `<video>` element does not force the browser to follow the value of this opt
 
     {{<Aside type="note">}}
 
-**Note:** Like all query string parameters, this value _must_ be URI encoded. For example, the thumbnail at `https://videodelivery.net/5d5bc37ffcf54c9b82e996823bffbb81/thumbnails/thumbnail.jpg?time=68s&height=270` can be encoded using JavaScript's `encodeURIComponent()` function to `https%3A%2F%2Fvideodelivery.net%2F5d5bc37ffcf54c9b82e996823bffbb81%2Fthumbnails%2Fthumbnail.jpg%3Ftime%3D68s%26height%3D270`.
+https://customer-<CODE>.iframe.cloudflarestream.com/<VIDEO_UID>
```
Comment (bkrebsbach) [inline content/stream/viewing-videos/using-the-stream-player.md]: customer-<CODE>.cloudflarestream.com/<VIDEO_ID>/iframe

**Review by bkrebsbach — CHANGES_REQUESTED** [review]
A couple of URLs need to be updated, otherwise looks good to me.

**Comment by irvinebroque** [comment]
Thanks for catching @bkrebsbach, ready for 👀

**Review by bkrebsbach — APPROVED** [review]

**Comment by bkrebsbach** [comment]
@irvinebroque LGTM!

**Pull request merged**